pkg/epforwarder/epforwarder.go

```diff
@@ -267,10 +278,11 @@ func (s *defaultEventPlatformForwarder) Stop() {
 }
 
 type passthroughPipeline struct {
-	sender   *sender.Sender
-	strategy sender.Strategy
-	in       chan *message.Message
-	auditor  auditor.Auditor
+	sender          *sender.Sender
+	strategy        sender.Strategy
+	in              chan *message.Message
+	auditor         auditor.Auditor
+	messageReceiver *diagnostic.BufferedMessageReceiver
```

Comment:
For consistency with the similar field in the logs pipeline, and to highlight that this isn't a regular part of the pipeline.

```suggestion
	diagnosticMessageReceiver *diagnostic.BufferedMessageReceiver
```